nexus/types/src/external_api/params.rs

```diff
@@ -1130,7 +1130,9 @@ pub enum ExternalIpCreate {
 #[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
 #[serde(tag = "type", rename_all = "snake_case")]
 pub struct EphemeralIpCreate {
-    /// Name or ID of the IP pool used to allocate an address
+    /// Name or ID of the IP pool used to allocate an address. The address is
+    /// automatically-assigned from the provided IP Pool, or the current silo's
```

Comment:
```suggestion
    /// automatically assigned from the provided IP pool, or the current silo's
```

Reply:
Actually, I think that first clause is a little redundant with the first sentence anyway. How about

> Name or ID of the IP pool used to allocate an address. If unspecified, the default pool will be used.

We try to avoid mentioning the silo if possible because for end users it is their entire universe anyway.

Reply:
I copied the text from `ExternalIpCreate` to `EphemeralIpCreate`. Updated in both.

Reply:
Happy to switch to that updated text. Want it in both places?

Reply:
Yes, thanks!